Sources/SwiftProtobuf/Data+Extensions.swift

```diff
@@ -32,3 +32,7 @@ internal extension Data {
     }
 }
 #endif
+
+#if swift(>=5.5) && !swift(>=6.0)
+extension Data: @unchecked Sendable {}
```

Comment:
Actually, should we really be globally adding to `Foundation`s `Data`?

Reply:
I don't think it's global, based on my reading, but let me generate some `swiftinterface` files and poke at them.

Reply:
I'd have though this might have needed to be public for a Swift 5.5 build with a Message containing a `bytes` field (`Data`) to be happy since you had to do extra work for the Storage cases.

Reply:
You are correct in your reasoning, but because `Sendable` doesn't have an ABI impact I'm not certain it has an observable impact outside of this module.

Reply:
Resolved by removal

Reply:
I was, though, able to confirm my hypothesis that two retroactive conformances to `Sendable` published by two separate modules doesn't break — _unlike_ normal Swift. We won't make use of that, but it's a neat lesson.